sdk/communication/communication-sms/rollup.base.config.js

```diff
@@ -41,7 +41,7 @@ export function nodeConfig(test = false) {
 
   if (test) {
     // Entry points - test files under the `test` folder(common for both browser and node), node specific test files
-    baseConfig.input = ["dist-esm/test/*.spec.js", "dist-esm/test/node/*.spec.js"];
+    baseConfig.input = ["dist-esm/test/**/*.spec.js"];
```

Comment:
I think I saw this pattern in another repo, but maybe that's a bad idea there too. This is because we don't currently have a browser folder for anything. That said, changing it for future proofing is not a bad idea